C22_Physics/SteveManager.h

```diff
@@ -0,0 +1,67 @@
+/*----------------------------------------------
+Programmer: Ruben Young (ray6190@rit.edu)
+Date: 2019/11
+----------------------------------------------*/
+#ifndef STEVEMANAGER_H
+#define STEVEMANAGER_H
+
+#include "Definitions.h"
+#include "LaneGrid.h"
+
+
+namespace Simplex{
+class SteveManager
+{
+    uint m_steveCount = 0;
+    LaneGrid* m_pLaneGrid = nullptr;
+
+public:
+    // Default Constructor and Destructor
+    SteveManager();
+    ~SteveManager();
+
+    void Init();
+
+    // This method handles:
+    // - Checking for dead Steves 
```

Comment:
I don't think this method really needs to check for dead steves. The Lane Grid will detect when the projectile should "explode" and will have a list of entities to delete that it will pass to the SteveManager through a function call.